src/Climbing.elm

```diff
@@ -6,8 +6,7 @@ import Element.Background as Background
 import Element.Border as Border exposing (rounded, widthXY)
 import Element.Font as Font exposing (center)
 import Element.Input as Input exposing (button, labelHidden, radioRow)
-import GetMatch exposing (getMatch)
-import TeamData exposing (stationToString, team)
+import TeamData
```

Comment:
Either `GetMatch`, `getMatch`, `stationToString` and `team` weren't used, or now `Climbing.elm` doesn't compile.

Reply:
they weren't used